pkg/cmd/cli/cli.go

```diff
@@ -69,6 +69,8 @@ func NewCommandCLI(name string) *cobra.Command {
 	cmds.AddCommand(cmd.NewCmdStartBuild(f, out))
 	cmds.AddCommand(cmd.NewCmdCancelBuild(f, out))
 
+	cmds.AddCommand(cmd.NewCmdRollback(name, "rollback", f, out))
```

Comment:
I'm not sure it should be called `rollback`, maybe `rollback-deployment` to follow the convention used in `start-build` and `cancel-build`. @smarterclayton thoughts?